streamer/Cargo.toml

```diff
@@ -21,7 +21,9 @@ nix = "0.24.2"
 pem = "1.0.2"
 percentage = "0.1.0"
 pkcs8 = { version = "0.8.0", features = ["alloc"] }
-quinn = "0.8.3"
+quinn = {git = "https://github.com/quinn-rs/quinn.git", branch = "0.8.x", commit = "37c19743cc881cf71369946d572849d5d2ffc3fd"}
+quinn-proto = {git = "https://github.com/quinn-rs/quinn.git", branch = "0.8.x", commit = "37c19743cc881cf71369946d572849d5d2ffc3fd"}
```

Comment:
@lijunwangs This does not seem correct. I am seeing warnings:
```
warning: solana/streamer/Cargo.toml: unused manifest key: dependencies.quinn-proto.commit                                                           
warning: solana/streamer/Cargo.toml: unused manifest key: dependencies.quinn.commit 
```

You need to use `rev` instead:
https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html#cargotoml-vs-cargolock
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

as in: 
https://github.com/solana-labs/solana/blob/773a4dd4d/Cargo.toml#L100

Reply:
Thanks. @behzadnouri -- will take care of it.